suite-native/accounts/src/components/TokenSelectBottomSheet.tsx

```diff
@@ -17,27 +20,30 @@ export const TokenSelectBottomSheet = ({
 }: TokenSelectBottomSheetProps) => {
     const [selectedAccount, setSelectedAccount] = useAtom(bottomSheetAccountAtom);
 
-    const handleSelectAccount: OnSelectAccount = params => {
+    const handleSelectAccount: OnSelectAccount = useCallback(
+        params => {
+            setSelectedAccount(null);
+            onSelectAccount(params);
+        },
+        [onSelectAccount, setSelectedAccount],
+    );
+
+    const handleClose = useCallback(() => {
         setSelectedAccount(null);
-        onSelectAccount(params);
-    };
+    }, [setSelectedAccount]);
 
+    const data = useSelector((state: NativeAccountsRootState) =>
+        selectAccountListSections(state, selectedAccount?.key),
+    );
     if (!selectedAccount) {
         return null;
     }
 
     return (
-        <BottomSheet
-            isVisible
-            onClose={() => {
-                setSelectedAccount(null);
-            }}
-        >
-            <AccountListItemInteractive
-                key={selectedAccount.key}
-                account={selectedAccount}
-                onSelectAccount={handleSelectAccount}
-            />
-        </BottomSheet>
+        <AccountSelectBottomSheet
```

Comment:
it due tohow it behaves for `isScrollable === false` it can now look like this. I am not sure whether this is expected:
<img width=300 src="https://github.com/user-attachments/assets/fa8a7c30-3f0b-43ae-8cd0-acc02b1342b3" />

Reply:
Probably yes, I decided that I will tweak that in another PR because it will require some extensive BottomSheet refactorings

Reply:
Okay. Some refactoring there might be useful. it does odd things with scrolling for no obvious reason (detecting pan instead of scroll).